step-capstone/src/components/Suggestions/SuggestionItem.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  CardHeader,
+  CardMedia,
+  Grid,
+  Typography
+} from '@material-ui/core'
+import FormPopover from "../TravelObjectForms/FormPopover"
+import "../../styles/Suggestions.css"
+import logo from "../../images/logo.png"
+
+export default function SuggestionItem(props) {
+
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+
+
+  }
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const printDollarSigns = (budget) => {
+    if (budget === undefined) {
+      return "";
+    } else if (budget === 0) {
+      return "Free";
+    } else {
+      let cost = "";
+      for (let i = 0; i < budget; i++) {
+        cost += "$";
+      }
+      return cost;
+    }
+  }
+
+  const suggestionToData = (suggestion) => {
+    return {
+      title: (props.suggestion.place.name === undefined) ? "" : props.suggestion.place.name,
+      placeId: props.suggestion.place_id,
+      finalized: props.context.finalized,
+      startDate: props.context.startDate,
+      endDate: props.context.endDate,
+      location: props.suggestion.place.vicinity,
+      coordinates: {
+        lat: props.suggestion.place.geometry.location.lat(),
+        lng: props.suggestion.place.geometry.location.lng()
+      } ,
+      description :"",
+      type: "event"
+    }
+  }
+
+  const open = Boolean(anchorEl);
+
+  return (
+    <Card className="suggestion-item" >
+      <CardHeader titleTypographyProps={{noWrap:"true"}} title={(props.suggestion.place.name === undefined) ? "" : props.suggestion.place.name} />
+
+      <CardMedia
+        className="suggestion-photo"
+        component="img"
+        image={(props.suggestion.place.photos === undefined) ? logo : props.suggestion.place.photos[0].getUrl()}
+        title="suggestion photo"
+      />
+
+      <CardContent>
+        <Grid container justify="space-between">  
+          <Typography className="suggestion-body" variant="subtitle2" inline gutterBottom align="left">
+            Price: {printDollarSigns(props.suggestion.place.price_level)} 
+          </Typography>
+          <Typography className="suggestion-body" variant="subtitle2" inline gutterBottom align="right">
+            Rating: {(props.suggestion.place.rating === undefined) ? "" : props.suggestion.place.rating}
```

Comment:
Stretch goal: Displaying the rating as a number of stars or dots out of five might be a bit more intuitive. Currently, I get a number value, but I don't know how high the max is so it doesn't necessarily mean much to me.

Reply:
or we can display the rating like this: 4.3 / 5 ?

Reply:
I think Dan's suggestion is pretty good for now. Places API might have some ui component that can give us a nice star-rating look (I know yelp does but not sure about Google)

Reply:
good ideas. for now, I have just set it to display /5 like Dan suggested. Stars and dots get hard with decimals like .3 so I will leave that as a stretch goal.